clash-prelude/src/Clash/Signal/Internal.hs

```diff
@@ -1026,26 +1028,31 @@ tbDynamicClockGen periods ena =
 --
 -- See 'Clash.Explicit.Testbench.tbClockGen' for example usage.
 --
+-- __NB__: While this can be used in the @testBench@ function, it cannot be
+-- synthesized to hardware.
```

Comment:
```suggestion
-- __NB__: While this can be used in test bench functions, it cannot be
-- synthesized to hardware.
```

Reply:
This seems to imply that any test bench is a [`TestBench`](https://hackage.haskell.org/package/clash-prelude-1.7.0/candidate/docs/Clash-Annotations-TopEntity.html#t:TopEntity). I prefer `testBench` for that reason.